ext/configuration.c

```diff
@@ -156,6 +157,51 @@ static bool dd_parse_tags(zai_str value, zval *decoded_value, bool persistent) {
     return true;
 }
 
+static bool dd_parse_http_error_statuses(zai_str value, zval *decoded_value, bool persistent) {
+    UNUSED(persistent);
```

Comment:
Please refer to the function above - it's forbidden to ignore the "persistent".

Reply:
taking a look